localisation/000_wc_traits.csv

```diff
@@ -638,3 +638,25 @@ light_elune_branch;Follower of Mother Moon;;Anh
 light_elune_branch_desc;This character worships the kind, generous and loving side of Elune. Elune's aspect known as the Mother Moon emphasizes tranquillity and harmony to allow for the greatest nurturing of Azeroth's creatures.;;Dieser Charakter verehrt die g�tige, gro�z�gige und liebevolle Seite von Elune. Elunes Aspekt, der als Mutter Mond bekannt ist, konzentriert sich auf Ruhe und Harmonie, damit die Kreaturen Azeroths bestens gedeihen k�nnen.;;;;;;;;;;;x
 dark_elune_branch;Follower of Night Warrior;;Anh�nger[From.GetinFemale] des Nachtkriegers;;;;;;;;;;;x
 dark_elune_branch_desc;This character worships the righteous, wroth and vindictive side of Elune. No one can hide from Elune's aspect known as the Night Warrior. She hunts down the villains and covers the blades of her justice with shadows.;;Dieser Charakter verehrt die rechtschaffene, zornige und rachs�chtige Seite von Elune. Niemand kann sich vor Elunes Aspekt, dem Nachtkrieger, verstecken. Sie bringt die Schurken zur Strecke und h�llt die Klingen ihrer Gerechtigkeit in Schatten.;;;;;;;;;;;x
+GetMonkNameCap_Monk;Monk;;M�nch;;;;;;;;;;;x
+class_monk_1;Worthless [GetMonkNameCap] I;;Wertlose[From.GetrMale] [GetMonkNameCap] I;;;;;;;;;;;x
+class_monk_2;Novice [GetMonkNameCap] II;;[GetMonkNameCap]-Novize II;;;;;;;;;;;x
+class_monk_3;Apprentice [GetMonkNameCap] III;;[GetMonkNameCap]-Lehrling III;;;;;;;;;;;x
+class_monk_4;Experienced [GetMonkNameCap] IV;;Erfahrene[From.GetrMale] [GetMonkNameCap] IV;;;;;;;;;;;x
+class_monk_5;Skilled [GetMonkNameCap] V;;Begabte[From.GetrMale] [GetMonkNameCap] V;;;;;;;;;;;x
+class_monk_6;Expert [GetMonkNameCap] VI;;[GetMonkNameCap]s-Experte VI;;;;;;;;;;;x
+class_monk_7;Great [GetMonkNameCap] VII;;Gro�e[From.GetrMale] [GetMonkNameCap] VII;;;;;;;;;;;x
+class_monk_8;Master [GetMonkNameCap] VIII;;Meister-[GetMonkNameCap] VIII;;;;;;;;;;;x
+class_monk_9;Grand Master [GetMonkNameCap] IX;;Gro�meister-[GetMonkNameCap] IX;;;;;;;;;;;x
+class_monk_10;Legendary [GetMonkNameCap] X;;Legend�re[From.GetrMale] [GetMonkNameCap] X;;;;;;;;;;;x
+GetMonkDesc_desc;This character is a monk. Masters of bare-handed combat, monks never rely solely on the need to have a weapon in their hands to defend against their enemies. Monks bring a unique martial arts style to any fight and harness an exotic form of magical energy that�s unfamiliar to those, who practice other arcane arts.;;Dieser Charakter ist ein M�nch. Als Meister des waffenlosen Kampfes verlassen sich M�nche niemals allein auf die Notwendigkeit, eine Waffe zur Verteidigung gegen ihre Feinde zur Hand zu haben. M�nche bringen einen einzigartigen Kampfkunststil in jeden Kampf ein und machen sich eine exotische Form der magischen Energie zunutze, die denjenigen, die andere arkane K�nste aus�ben, unbekannt ist.;;;;;;;;;;;x
```

Comment:
Removed unnecessary comma and apostrophe in last sentence. Added an `is` in place of the apostrophe.
```suggestion
GetMonkDesc_desc;This character is a monk. Masters of bare-handed combat, monks never rely solely on the need to have a weapon in their hands to defend against their enemies. Monks bring a unique martial arts style to any fight and harness an exotic form of magical energy that is unfamiliar to those who practice other arcane arts.;;Dieser Charakter ist ein M�nch. Als Meister des waffenlosen Kampfes verlassen sich M�nche niemals allein auf die Notwendigkeit, eine Waffe zur Verteidigung gegen ihre Feinde zur Hand zu haben. M�nche bringen einen einzigartigen Kampfkunststil in jeden Kampf ein und machen sich eine exotische Form der magischen Energie zunutze, die denjenigen, die andere arkane K�nste aus�ben, unbekannt ist.;;;;;;;;;;;x
```